src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,369 @@
+be used to complete a payment.
+
+Payment receivers who wish to initiate an Easy Checkout flow MUST query the sender's PayID Discovery server to 
+obtain a PayID Easy Checkout URL. PayID servers that wish to enable PayID Easy
+Checkout MUST include a JRD Link conforming to the definition in the [JRD section](#payid-easy-checkout-jrds) of this paper 
+in all PayID Easy Checkout Discovery responses.
+
+Payment receivers SHOULD implement fallback measures to complete a checkout flow if a user's wallet does not support PayID Easy Checkout.
+
+The following steps describe how a PayID client can query a PayID server to obtain a PayID Easy Checkout URL. 
+
+### Step 1: Assemble PayID Easy Checkout Discovery URL
+The process of assembling a PayID Discovery URL is defined in section 4.1.1 of [PAYID-DISCOVERY][], and is
+the same as for this protocol.
+
+### Step 2: Query PayID Easy Checkout Discovery URL
+Querying the PayID Discovery URL is defined in section 4.1.2 of [PAYID-DISCOVERY][], and is performed
+in the same way as this protocol.
+
+Clients SHOULD implement fallback measures to complete checkout if the PayID Easy Checkout Discovery query fails.
+
+### Step 3: Parse PayID Easy Checkout Metadata
+If PayID Easy Checkout is supported, a PayID Discovery server MUST respond with an HTTP status code `200` and a JSON payload
+containing a JSON Resource Descriptor (JRD) that contains a link conforming to the [JRD section](#payid-easy-checkout-jrds) of this document.
+
+For example, a PayID server might respond to a PayID Discovery query with the following payload:
+
+     {
+        "subject": "payid:alice$wallet.com",
+        "links": [
+            {
+                "rel" : "https://payid.org/ns/payid-easy-checkout/1.0",
+                "href": "https://wallet.com/checkout"
+            }
+        ]
+     }
+     
+A PayID Discovery client MUST parse this response to find the PayID Easy Checkout Link. 
+If the JRD returned from the PayID Discovery query does not contain a 
+PayID Easy Checkout Link in its 'links' collection, PayID Easy Checkout is considered to have failed.
+Once a PayID Easy Checkout URL has been obtained from the PayID Easy Checkout Link, 
+PayID Easy Checkout Discovery is considered to be complete. 
+
+## PayID Easy Checkout URL Assembly
+A PayID Easy Checkout URL represents the resource on a wallet that can
+be used by a sender to complete a payment. However, before directing a payer to their sending client, the payment receiver
+MUST append all of the query parameters defined in the [following section](#payid-easy-checkout-url-query-parameters).
+
+Once a PayID Easy Checkout URL is assembled, PayID Easy Checkout is considered to be complete.
+
+### PayID Easy Checkout URL Query Parameters
+This specification defines several query parameter names and corresponding datatypes which MUST be added to the
+PayID Easy Checkout URL before redirecting a payer to their wallet client. The PayID Easy Checkout URL SHOULD be parsed 
```

Comment:
Why is the reciever's adding the query params a MUST, but the wallet parsing them is only a SHOULD?

Reply:
The query params are meant to be the lowest common denominator for wallets so that PayID Easy Checkout clients (merchants) don't need to handle different logic for different wallets, hence the MUST.  Wallets parsing them don't necessarily need to use all of the params. For example, if you are paying with Xumm, your Xumm app probably doesnt need to look at `paymentNetwork`, since it's only capable of sending over XRPL.  Another example - Imagine you want to donate to a charity with Easy Checkout.  The charity might have a "suggested" donation amount, which they could put in the URL, but you could override that in your wallet if you wanted to send less or more.  

Bottom line: we want to provide enough info for all wallets to be able to send a transaction, but this protocol doesn't really care what individual wallets do with that information.